internal/cache/canvases.go

```diff
@@ -2,13 +2,14 @@ package cache
 
 import (
 	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/internal/async"
 )
 
-var canvases = make(map[fyne.CanvasObject]*canvasInfo, 1024)
```

Comment:
Can we put a comment here that this is an async.Map just for compatibility reasons and should be switched back to regular map at 3.0 or whenever we decide to fully run down the threading migration plan?

Reply:
I don't think that is necessary. I reverted all my other async.Map removals so there is basically a lot more to move over and it is tracked in the migration plan.